src/dlgTriggerEditor.cpp

```diff
@@ -179,39 +179,34 @@ dlgTriggerEditor::dlgTriggerEditor( Host * pH )
     connect(mpScriptsMainArea->listWidget_registered_event_handlers, SIGNAL(itemClicked ( QListWidgetItem * )), this, SLOT(slot_script_main_area_edit_handler(QListWidgetItem*)));
 
     // source editor area
-
     mpSourceEditorArea = new dlgSourceEditorArea( mainArea );
     QSizePolicy sizePolicy5(QSizePolicy::Expanding, QSizePolicy::Expanding);
     mpSourceEditorArea->setSizePolicy( sizePolicy5 );
     pVB1->addWidget( mpSourceEditorArea );
 
     // And the new edbee widget - Go Buck!
-
     mpSourceEditorEdbee = mpSourceEditorArea->edbeeEditorWidget;
     mpSourceEditorEdbeeDocument = mpSourceEditorEdbee->textDocument();
 
-    // Updating the status bar on changes
-
+    // Update the status bar on changes
     connect( mpSourceEditorEdbee->controller(),
              SIGNAL(updateStatusTextSignal(QString)),
              this,
              SLOT(slot_updateStatusBar(QString))
          );
     simplifyEdbeeStatusBarRegex = new QRegularExpression(R"(^(?:\[\*\] )?(.+?) \|)");
 
-    // Updating the editor preferences
-
+    // Update the editor preferences
     connect( mudlet::self(),
              SIGNAL(signal_editorTextOptionsChanged(QTextOption::Flags)),
              this,
              SLOT(slot_changeEditorTextOptions(QTextOption::Flags))
          );
 
-    mpSourceEditorEdbee->config()->setShowWhitespaceMode( mudlet::self()->mEditorTextOptions & QTextOption::ShowTabsAndSpaces);
-    mpSourceEditorEdbee->config()->setUseLineSeparator( mudlet::self()->mEditorTextOptions & QTextOption::ShowLineAndParagraphSeparators);
-
     mpSourceEditorEdbeeDocument->setText( QString("# Enter your lua code here\n"));
 
+    mudlet::loadEdbeeTheme(mpHost->mEditorTheme, mpHost->mEditorThemeFile);
```

Comment:
Shouldn't this be done just once when mudlet loads, instead of once per loaded profile?